components/global/global_header.scss

```diff
@@ -47,6 +53,43 @@
             }
         }
 
+        .channel__wrap {
+
+            // indicates the RHS is open
+            &.move--left {
+
+                // adjust channel header width
+                #channel-header {
+                    width: calc(100% - 400px);
+                }
+            }
+        }
+
+        // adjust RHS position and height
+        #sidebar-right {
+            z-index: 20;
+            top: 0;
+
+            .sidebar-right-container {
+                height: 100%;
+            }
+
+            .sidebar--right__header {
+                flex-basis: 63px;
```

Comment:
I am not 100% sure, but I know that some browsers have different default values for flex. It would probably be a good idea (to prevent issues) to set all values for the flex property.

Reply:
And isn't 63px restricting the width as well? Just wondering.

Reply:
Good thoughts! I don't generally use `flex-basis`, but in this case needed to override an existing applied style of `flex: 0 0 56px;` and so it should be fine to use `flex-basis: 64px;` directly. `flex-basis` controls the size in the defined `flex-direction` and not both directions so I think we're good.

Although it's not very intuitive by name (IMO), I guess `flex-basis` is helpful for situations where a `flex` container could use either `flex-direction: row` or `flex-direction: column` and where you need the same width or height respectively for either direction. I would normally use width/height otherwise.

Reply:
yeah ... flex can be qwirky at times. All good ... those were just thoughts on possible problems anyways.